common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -1189,13 +1198,14 @@ def test_get_items(self):
         locator = CourseLocator(org='testx', course='GreekHero', run="run", branch=BRANCH_NAME_DRAFT)
         # get all modules
         matches = modulestore().get_items(locator)
-        self.assertEqual(len(matches), 7)
+        self.assertEqual(len(matches), 8)
         matches = modulestore().get_items(locator)
-        self.assertEqual(len(matches), 7)
+        self.assertEqual(len(matches), 8)
         matches = modulestore().get_items(locator, qualifiers={'category': 'chapter'})
-        self.assertEqual(len(matches), 3)
+        self.assertEqual(len(matches), 4)
         matches = modulestore().get_items(locator, qualifiers={'category': 'garbage'})
         self.assertEqual(len(matches), 0)
+        # Test that we don't accidentally get an item with a similar name.
         matches = modulestore().get_items(locator, qualifiers={'name': 'chapter1'})
         self.assertEqual(len(matches), 1)
```

Comment:
It goes the other way around - currently, searching for a course block with a "long" id (for example, `chapter1`) will match all blocks with names that are substrings of that name (for example, `chap`). So here, the test is more about adding a course block whose name is a substring to the data we're working with, and then verifying that the behavior in the existing test remains the same for the case where we search for the "long" name; in other words, that the addition of a block that, previously, would have messed with the result, doesn't.